### DISCUSSION TITLE
How do you pronounce 'pinia'

### DISCUSSION THREAD

**Discussion by bigyifeng** [vuejs/pinia#1928] [Help and Questions]
sorry ！I don't know how to say.

**Comment by posva** [comment]
https://pinia.vuejs.org/introduction.html#why-pinia

> Pinia (pronounced /piːnjʌ/, like "peenya" in English) is the closest word to piña (pineapple in Spanish) that is a valid package name

Not that I personally care how people pronounce it as I picked it because it looked very similar no matter the language but if you are curious about the actual sound [take a look at the Spanish sound](https://www.youtube.com/watch?v=WOI5tLXmcfg) of the word pineapple piña
